client/src/components/App.js

```diff
@@ -9,7 +9,7 @@ import QA from './QA';
 
 function App({ initStore }) {
   useEffect(() => {
-    initStore(20);
+    initStore(55);
```

Comment:
Maybe don't include `App.js` if change id for testing purpose?

Reply:
Fixed!